forch/port_state_manager.py

```diff
@@ -107,7 +109,8 @@ class PortStateManager:
 
     # pylint: disable=too-many-arguments
     def __init__(self, process_device_placement, process_device_behavior, get_vlan_from_segment,
-                 update_device_state_varz=None, testing_segment=None):
+                 update_device_state_varz=None, update_static_vlan_varz=None,
```

Comment:
this pattern (argument list of callback functions) it's right -- it's starting to collect a bunch of things in it and it's not clean code. We should think a bit about the abstraction here and make sure we're implementing all of this correctly.  It might mean that we need another object/class that encapsulates some of this behavior and pass that one class in, rather than a bunch of separate callbacks!

Right now, there are ~5 functions here being passed in... how many different target objects do they go to?